Packs/Forcepoint/Scripts/FPDeleteRule/FPDeleteRule.py

```diff
@@ -0,0 +1,54 @@
+import sys
+from re import escape
+
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+
+FILTER_CONFIG_PATH = "/opt/WCG/config/filter.config"
+CMD_DEL_RULE_FORMAT = "sed -i '/^{0}={1} action=[A-Za-z]*$/d' {2}"
+CMD_TRITON_RELOAD_CONFIG = "/opt/WCG/bin/content_line -x"  # && /opt/WCG/WCGAdmin runds"
+
+
+def delete_rule(ruleType):
+    if ruleType not in ["dest_domain", "dest_ip", "dest_host", "url_regex"]:
```

Comment:
Create a constant with the rule types, and use it here and in the error message.

Reply:
Why do we need to do so many changes in the code? 
I already added main, function, and unit tests to increase coverage from 0 to 70, why more effort is needed? 
It is not related to python2 to 3 issues, And if you want the code to be as more as elegant there are a lot of changes that should be done, but these changes are not likely to happen since we have no instances.

Reply:
@dantavori